aten/src/ATen/native/native_functions.yaml

```diff
@@ -515,15 +515,15 @@
 
 - func: asinh(Tensor self) -> Tensor
   variants: function, method
-  dispatch:
-    CPU, CUDA: asinh
+  structured_delegate: asinh.out
 
 - func: asinh_(Tensor(a!) self) -> Tensor(a!)
   variants: function, method
-  dispatch:
-    CompositeExplicitAutograd: asinh_
+  structured_delegate: asinh.out
 
 - func: asinh.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
+  structured: True
+  structured_inherits: TensorIteratorBase
```

Comment:
curious: what does this mean?

Reply:
https://github.com/pytorch/rfcs/blob/rfc-0005/RFC-0005-structured-kernel-definitions.md#handling-tensoriterator